LibAddonMenu-2.0/LibAddonMenu-2.0/LibAddonMenu-2.0.lua

```diff
@@ -344,9 +344,9 @@ local localization = {
         PANEL_NAME = "Addon",
         VERSION = "Versione: <<X:1>>",
         WEBSITE = "Visita il Sitoweb",
-        RELOAD_UI_WARNING = "Cambiare questa impostazione richiede un Reload UI al fine che faccia effetto.",
+        RELOAD_UI_WARNING = "Cambiare questa impostazione richiede un Ricarica UI al fine che faccia effetto.",
         RELOAD_DIALOG_TITLE = "Reload UI richiesto",

```

Comment:
Are you sure you want to leave it like that when you changed the other two lines?